src/Notion.ts

```diff
@@ -495,6 +498,39 @@ export class Notion {
     });
   }
 
+  /**
+ * Activates haptic motors
+ * 
+ * ```typescript
+ * notion.haptics({P7: ["tripleClick100"], P8: ["tripleClick100"]});
```

Comment:
It would be great to mention that the motor names are based on the 1-20 EEG system.